package.json

```
@@ -12,31 +13,35 @@
     "README.md"
   ],
   "scripts": {
+    "build-parser": "Make",
```

```suggestion
    "build-parser": "make parser",
```

Be explicit whenever possible in scripts. Shortcuts and defaults are for use by humans.